content/en/docs/components/pipelines/v2/author-a-pipeline/pipelines.md

```diff
@@ -68,6 +68,55 @@ def my_pipeline():
         train_model(epochs=epochs)
 ```
 
+### dsl.Collected
+
+Use [`dsl.Collected`](https://kubeflow-pipelines.readthedocs.io/en/latest/source/dsl.html#kfp.dsl.Collected) with `dsl.ParallelFor` to gather outputs from a parallel loop of tasks:
+
+```python
+from kfp import dsl
+
+@dsl.pipeline
+def my_pipeline():
+    with dsl.ParallelFor(
+        items=[1, 5, 10, 25],
+        parallelism=2
+    ) as epochs:
+        train_model_task = train_model(epochs=epochs)
+    max_accuracy(models=dsl.Collected(train_model_task.outputs['model']))
+```
+
+Downstream tasks may consume `dsl.Collected` outputs via an input annotated with a `List` of parameters or `List` of artifacts. For example, `max_accuracy` in the above example has the input `models` with type `List[Model]`:
+
+```python
+from kfp import dsl
+from kfp.dsl import Model, Output
+
+@dsl.component
+def select_best(models: List[Model]) -> float:
+    return max(score_model(model) for model in models)
+```
+
+`dsl.Collected` can be used to collect outputs from nested loops as well.
+
+Collection of *parameters* from nested loops will result in a *nested list* of parameters (e.g., two nested `dsl.ParallelFor` groups results in a **list of lists** of parameters). By comparison, collection of *artifacts* from nested loops results in a *flattened* lists of artifacts (e.g., any number of nested `dsl.ParallelFor` groups results in a **single list** containing all artifacts).
+
+You may also return a `dsl.Collected` from a pipeline using a `List` of parameters or `List` of artifacts return annotation:
```

Comment:
Similar comment as above - replace "may" with "might", add "a" after "or".

Is "artifacts return annotation" a single term?

Reply:
Updated.

It's not really a single term. "`List` of artifacts" is one thing and the "return annotation" is how it's used. I updated this slightly for clarity.